src/main/java/com/coveros/selenified/element/Element.java

```diff
@@ -1111,84 +1093,85 @@ public void selectValue(String value) {
             dropdown.selectByValue(value);
         } catch (Exception e) {
             log.warn(e);
-            file.recordStep(action, expected, CANT_SELECT + prettyOutput() + ". " + e.getMessage(), Success.FAIL);
-            file.addError();
+            reporter.fail(action, expected, CANT_SELECT + prettyOutput() + ". " + e.getMessage());
             return;
         }
-        file.recordStep(action, expected, "Selected <b>" + value + INN + prettyOutputEnd(), Success.PASS);
+        reporter.pass(action, expected, "Selected <b>" + value + INN + prettyOutputEnd());
     }
 
     /**
      * Generates and logs an error (with a screenshot), stating that the element
-     * was unable to me moved to
+     * was unable to be scrolled to
      *
      * @param e        - the exception that was thrown
      * @param action   - what is the action occurring
      * @param expected - what is the expected outcome of said action
      */
-    private void cantMove(Exception e, String action, String expected) {
+    private void cantScroll(Exception e, String action, String expected) {
         log.warn(e);
-        file.recordStep(action, expected, CANT_MOVE + prettyOutput() + ". " + e.getMessage(), Success.FAIL);
-        file.addError();
+        reporter.fail(action, expected, CANT_SCROLL + prettyOutput() + ". " + e.getMessage());
     }
 
     /**
-     * Determines if the element moved towards is now currently displayed on the
+     * Determines if the element scrolled towards is now currently displayed on the
      * screen
      *
      * @param action   - what is the action occurring
      * @param expected - what is the expected outcome of said action
      */
-    private void isMoved(String action, String expected) {
-        if (!is.displayed()) {
-            file.recordStep(action, expected, prettyOutputStart() + " is not displayed within the current viewport",
-                    Success.FAIL);
-            file.addError();
-            return; // indicates element not on displayed screen
+    private void isScrolledTo(String action, String expected, long offset) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "offset". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1172)